doc/performance-testing/cloud-run/0001_stress-test-december-2021.md

```diff
@@ -0,0 +1,86 @@
+# Stress Test December 2021
+This is an initial stress test to asses the performance of runner after the switch from kubernetes to cloud run. Below are the initial settings we used:
```

Comment:
For me because you are not including the comparable results from kubernetes, this should just be written up as just a runner test using cloud run. I also don't think you need the initial bit, it does makes me feel there is more to come on this write up

Reply:
Removed this now 👍